### ISSUE TITLE
New posts icon is shown even if the newest post is mine

### ISSUE THREAD

**Issue by vadosnaprimer** [TASVideos/tasvideos#1341]
![изображение](https://user-images.githubusercontent.com/7092625/173881450-3ea905b9-640b-4c47-9ebc-7f901d9b1a54.png)

![изображение](https://user-images.githubusercontent.com/7092625/173881844-15eb68e3-4f6a-472d-8d9a-c0d43925793a.png)

It would make sense to clear the flag after you've posted in a thread, because it kinda counts as having visited it.

**Comment by Masterjun3** [comment]
Yeah so I can't recreate this. But I can see how this can occur.

The system currently works like this:
Every time you visit the last page of a topic page, your browser executes some javascript to save the current timestamp you visited the topic. So when you make a post you get redirected to your post and automatically save the timestamp.
Now, in the topic view, the server *always* sends you the new-post icon for the topics and the timestamp of it, and your browser decides whether to hide based on the saved timestamp.

So basically if your operating system time is off by over 1 second from the server time, this can happen.

And I'm not sure if we can do something about this. If we increase the tolerance then it's possible you visit a topic and a few seconds later someone makes a post and you will never see an icon.
